ArjunaJTA/recovery/src/main/java/org/jboss/narayana/jta/quickstarts/recovery/JmsRecovery.java

```diff
@@ -172,35 +169,18 @@ private static void startActiveMQServer(String acceptorName, String connFacName)
 
     // Initialise client side objects: connection factory and JMS queue
     private static void initialiseActiveMQClient(String connFacName) {
-        xacf = ActiveMQJMSClient.createConnectionFactoryWithoutHA(JMSFactoryType.XA_CF, new TransportConfiguration(connFacName));
-        try(Session s = xacf.createConnection().createSession()) {
+        xacf = ActiveMQJMSClient.createConnectionFactoryWithoutHA(JMSFactoryType.XA_CF,
+                new TransportConfiguration(connFacName));
+        try (Session s = xacf.createConnection().createSession()) {
```

Comment:
Yep, I see your point here. My only concern is whether this is a functional or a style change. I'd lean toward functional here but that's up to you

Reply:
I understand, I split the commits now. This change will be included in the "Cleaning the code" commit. Thanks